### PR TITLE
Fixing security configuration tests

### PR DESCRIPTION
**Pull request by honza-kasik**
Added Creaper to tests, fixed module options checking, cleaning after tests, trying to handle resources better.

### CODE REVIEW

**Comment by honza-kasik** [comment]
Please wait with merging!

**Comment by honza-kasik** [comment]
Ready to be merged! I was just looking to diff and big blocks of removed code scared me a bit. But I checked it and no tests were removed :).

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/security/ACLTestCase.java [hunk 1]
```diff
@@ -3,62 +3,62 @@
...
- *         Created on 16.10.15.
- */
 @RunWith(Arquillian.class)
 @Category(Shared.class)
 public class ACLTestCase extends SecurityTestCaseAbstract {
 
-    private static final AddressTemplate ACL_TEMPLATE = SECURITY_DOMAIN_TEMPLATE.append("acl=classic/acl-module=*");
-    private static ResourceAddress ACL_ADDRESS;
-    private static String aclModule;
+    private static final Address ACL_ADDRESS = JBOSS_WEB_ADDRESS.and("acl","classic");
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/security/ACLTestCase.java]: Insert space before `"classic"` to comply with checkstyle rules and run at least `mvn clean test -Pbasic,rbac -Djboss.dist -DskipTests` next time.

**Comment by honza-kasik** [comment]
Fixed style, sorry.

**Comment by zebrik** [comment]
LGTM. Thank you.

**Pull request merged**